src/styles/header.css

```diff
@@ -0,0 +1,6 @@
+.header {
+  margin-top: 40px;
```

Comment:
'header' is your block. We can not give external styling like margin or positioning to block. You can create an element and then give it the external styling.
refer checklist number 6

Reply:
Moved margin to element style.

Reply:
Deleted this block

Reply:
and moved header to card block